calculation/seistech_calc/seistech_calc/plots/plt_plotting.py

```diff
@@ -549,3 +592,403 @@ def plot_uhs_branches(
         plt.close()
     else:
         plt.show()
+
+
+def plot_gms_im_distribution(
+    gms_result_data: Dict,
```

Comment:
Also I suggest using the GMS result objects directly instead of dictionary derived from it. (I know the other plotting functions also do this, however since the plots are now part of `seistech_calc` and are only going to be used with `seistech_calc` result objects, we should switch to just working on those. )

Sorry, probably should have realised that sooner..